### ISSUE TITLE
Evil Blog Integration

### ISSUE THREAD

**Issue by pablopeu** [tkuther/piwigo-bootstrap-darkroom#57]
Hi, is there a chance to add the extension Evil-Blog to the list of compatible plugins for Piwigo? Not that I'm a big fan of it, its the only blog plugin I found... :)

**Comment by tkuther** [comment]
Hi Pablo,

did you try it? I could image it does work mostly and just needs some styling...

**Comment by pablopeu** [comment]
Thomas, its definitely not wordpress :) but it allows you to create an HTML entry. I just enabled it for a moment and how it looks not integrated to darkroom is awful.

If you use my site as an example, a good location to see the entry header and a 3 line preview would be below the categories photos, with a configurable limitation on the amount of entries that can be seen on main page.

Maybe its too basic to be integrated into an elegant theme such as darkroom... and it should have its own top level menu entry

**Comment by pablopeu** [comment]
So, this plugin could become a supported one? Thanks!

**Comment by pablopeu** [comment]
I really miss the chance of having a Blog at my gallery... Sadly I'm not a programmer, just a knifemaker!

**Comment by m-rufflesmcghie** [comment]
Sorry to comment on issue from year ago, did you manage to get this working ?? I'm after a blog style for piwigo want to avoid using wordpress if possible.

**Comment by pablopeu** [comment]
Nope, found no way to integrate a blog into piwigo and I'm on the same boat
as you, I don't want to use wordpress

**Comment by m-rufflesmcghie** [comment]
https://piwigo.org/doc/doku.php?id=user_documentation:learn:customize:gallery_blog_additional_pages

Have you seen this ? Seems to work ok for said site but not sure if it's wp or piwigo...all the document links are outdated too :-(

**Comment by pablopeu** [comment]
That site uses wordpress now, there must be a reason LOL
